HPXMLtoOpenStudio/tests/test_hvac_sizing.rb

```diff
@@ -381,6 +382,101 @@ def test_allow_increased_fixed_capacities
+        hpxml_bldg.heat_pumps[0].cooling_capacity = nil
+        XMLHelper.write_file(hpxml.to_doc, @tmp_hpxml_path)
+        _model, _hpxml, hpxml_bldg = _test_measure(args_hash)
+        htg_cap_orig = hpxml_bldg.heat_pumps[0].heating_capacity
+        clg_cap_orig = hpxml_bldg.heat_pumps[0].cooling_capacity
+        backup_htg_cap_orig = hpxml_bldg.heat_pumps[0].backup_heating_capacity
+        # apply autosizing factor
+        hpxml, hpxml_bldg = _create_hpxml('base-hvac-air-to-air-heat-pump-1-speed.xml')
+        hpxml_bldg.heat_pumps[0].backup_heating_capacity = nil
+        hpxml_bldg.heat_pumps[0].heating_capacity = nil
+        hpxml_bldg.heat_pumps[0].cooling_capacity = nil
+        hpxml_bldg.heat_pumps[0].heating_autosizing_factor = haf
+        hpxml_bldg.heat_pumps[0].cooling_autosizing_factor = caf
+        XMLHelper.write_file(hpxml.to_doc, @tmp_hpxml_path)
+        _model, _hpxml, hpxml_bldg = _test_measure(args_hash)
+        htg_cap = hpxml_bldg.heat_pumps[0].heating_capacity
+        clg_cap = hpxml_bldg.heat_pumps[0].cooling_capacity
+        assert_in_epsilon(hpxml_bldg.heat_pumps[0].heating_capacity, htg_cap_orig * haf, 0.001)
+        assert_in_epsilon(hpxml_bldg.heat_pumps[0].cooling_capacity, clg_cap_orig * caf, 0.001)
+        # Fixme: backup heating not impacted, is it right?
+        assert_equal(hpxml_bldg.heat_pumps[0].backup_heating_capacity, backup_htg_cap_orig)
+
+        # Test heat pump w/ detailed performance
+        hpxml, hpxml_bldg = _create_hpxml('base-hvac-air-to-air-heat-pump-var-speed-detailed-performance-autosize.xml')
+        hpxml_bldg.heat_pumps[0].backup_heating_capacity = nil
+        XMLHelper.write_file(hpxml.to_doc, @tmp_hpxml_path)
+        _model, _hpxml, hpxml_bldg = _test_measure(args_hash)
+        htg_cap_orig = hpxml_bldg.heat_pumps[0].heating_capacity
+        clg_cap_orig = hpxml_bldg.heat_pumps[0].cooling_capacity
+        backup_htg_cap_orig = hpxml_bldg.heat_pumps[0].backup_heating_capacity
+        # apply autosizing factor
+        hpxml, hpxml_bldg = _create_hpxml('base-hvac-air-to-air-heat-pump-var-speed-detailed-performance-autosize.xml')
+        hpxml_bldg.heat_pumps[0].backup_heating_capacity = nil
+        hpxml_bldg.heat_pumps[0].heating_capacity = nil
+        hpxml_bldg.heat_pumps[0].cooling_capacity = nil
+        hpxml_bldg.heat_pumps[0].heating_autosizing_factor = haf
+        hpxml_bldg.heat_pumps[0].cooling_autosizing_factor = caf
+        XMLHelper.write_file(hpxml.to_doc, @tmp_hpxml_path)
+        _model, _hpxml, hpxml_bldg = _test_measure(args_hash)
+        htg_cap = hpxml_bldg.heat_pumps[0].heating_capacity
+        clg_cap = hpxml_bldg.heat_pumps[0].cooling_capacity
+        assert_in_epsilon(hpxml_bldg.heat_pumps[0].heating_capacity, htg_cap_orig * haf, 0.001)
+        assert_in_epsilon(hpxml_bldg.heat_pumps[0].cooling_capacity, clg_cap_orig * caf, 0.001)
+        # Fixme: backup heating not impacted, is it right?
```

Comment:
@jonwinkler Hi Jon, here's a quick question for you, if heating autosizing factor is defined, should it also apply to backup heating capacities?

Reply:
> @jonwinkler Hi Jon, here's a quick question for you, if heating autosizing factor is defined, should it also apply to backup heating capacities?

@yzhou601 Good question...I would think so, only because it might be easier for the user to understand the implementation.

Reply:
I have to admit that I find this approach confusing. For a heat pump, if you want it sized larger than the autosizing algorithm produces to reduce backup energy use, what is the point of also upsizing the backup? The backup is already sized to meet the entire heating design load.

Personally, I think we should have a separate `BackupHeatingAutosizingFactor` that defaults to 1.

Reply:
Having a separate input for backup autosizing factors sounds good to me, not confusing users while keeping the approach clean.